codegen/datamodelgen.py

```diff
@@ -121,6 +121,9 @@ def __init__(self):
             "preferences": DataModelStaticInfo(
                 "preferences", ("meshing", "solver", "flicing,"), self.version
             ),
+            "solver_workflow": DataModelStaticInfo(
+                "solverworkflow", ("solver",), self.version
```

Comment:
@seanpearsonuk, WE want to implement "solverworkflow" as "solver_workflow" in pyfluent?

Reply:
@prmukherj no, we ought to align it with what's in the Python console, so "solverworkflow".

Reply:
@seanpearsonuk, Thank you